packages/api-kit/tests/utils/setupKits.ts

```diff
@@ -0,0 +1,61 @@
+import Safe, { SafeProviderConfig, Eip1193Provider } from '@safe-global/protocol-kit'
+import SafeApiKit from '@safe-global/api-kit/index'
+import { ethers, web3 } from 'hardhat'
+
+import config from './config'
+
+type GetKits = {
+  protocolKit: Safe
+  safeApiKit: SafeApiKit
+}
+
+type GetKitsOptions = {
+  signer?: SafeProviderConfig['signer']
+  txServiceUrl?: string
+  safeAddress: string
+}
+
+function getEip1193Provider(): Eip1193Provider {
+  switch (process.env.ETH_LIB) {
```

Comment:
Could we add Viem here too?

Reply:
yes i can. Anyway in the github action only the ethers script it's being executed. Do you think is worth to add the matrix execution as in the protocol-kit?

Reply:
Ok i added viem and some more things as removing the hardhat web3 specific library, renaming the jobs both filenames and internally because it's never clear what each action does. I'm executing the e2e in the api-kit for the 3 libraries but couldn't do it in parallel (matrix) because some of them fail. In sequence seems to work